main.tf

```diff
@@ -11,8 +11,8 @@ locals {
 
   crypto_key_id = var.database_encryption_key_name == null ? google_kms_crypto_key.crypto_key[0].id : var.database_encryption_key_name
 
-  #  webhooks: prometheus-adapter, hephaestus, istio
-  required_webhooks = ["6443", "9443", "15017"]
+  #  webhooks: prometheus-adapter, hephaestus, workload-service, istio
```

Comment:
It is actually the vault agent injector that needs this, not workload service.

Reply:
ah! you're right, nice catch!